src/rdb/rdb_raft.c

```diff
@@ -1673,6 +1751,11 @@ struct rdb_raft_state {
 	uint64_t	drs_committed;
 };
 
+/* kccain - raft calls:
+ *  raft_is_leader()
+ *  raft_get_current_term()
+ *  raft_get-commit_idx()
+ */
 /* Save the variables into "state". */
 static void
 rdb_raft_save_state(struct rdb *db, struct rdb_raft_state *state)
```

Comment:
Yep, this function shall be called with d_raft_mutex held (or it shall take d_raft_mutex itself).